src/tasks/qa.py

```diff
@@ -0,0 +1,326 @@
+"""Task definitions for question answering tasks."""
```

Comment:
Do you know why this is showing up as a diff? Most of this isn't new, right?

Reply:
Ah, I see. Your changes are in src/, but the files in the repo are in jiant/. You'll have to move your changes over manually.

Reply:
Should be fixed now.